packages/block-library/src/cover/editor.scss

```diff
@@ -36,6 +39,14 @@
 	margin-left: auto;
 }
 
+.block-library-cover__resize-container {
```

Comment:
It's not strictly related to this PR, but we could update this class name to `.wp-block-cover__resize-container` for consistency.

Reply:
No, we shouldn't, this class follows our CSS guidelines already and the `wp-something-block**` class names are only for frontend classNames.